services/account_type/master_payer.yaml

```diff
@@ -15,6 +15,12 @@ Parameters:
     AllowedValues:
       - 'true'
       - 'false'
+  IsOrganizationMasterAccount:
```

Comment:
NOTE: Need this for CFN "api", that is parent stack expects this in all child stacks even if it's not used